guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
 
+# Mews Certification
+
+Before going live in the production environment all connections must be certified by Mews.
+
+### Pre-Certification
+
+- Sign in to the Mews Connector API demo account. Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section.
```

Comment:
Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section. -> Follow instruction... ?

Reply:
Adjusted to match the link to the authentication in guidelines.